### PR TITLE
Updates and bug fixes.

### PR DESCRIPTION
**Pull request by michaelawyu**

### CODE REVIEW

File: auth/index.js [hunk 1]
```diff
@@ -37,10 +37,12 @@ const setUpGmailPushNotifications = (email, pubsubTopic) => {
 // If the authorization process completes successfully, set up Gmail push
 // notification using the tokens returned
 const onSuccess = async (req, res) => {
+  var email;
```
Comment (ace-n) [inline auth/index.js]: Use `let`?

File: pubsub/index.js [hunk 2]
```diff
@@ -68,35 +74,46 @@ const extractInfoFromMessage = (message) => {
...
+  var topLabels = ['', '', ''];
+  if (filename.endsWith('.png') || filename.endsWith('.jpg')) {
+    const [analysis] = await visionClient.labelDetection({
+      image: {
+        content: Buffer.from(data, 'base64')
+      }
+    });
+    const labels = analysis.labelAnnotations;
+    for (var i = 0; i <= 2; i++) {
+      if (labels[i] && labels[i].description) {
```
Comment (ace-n) [inline pubsub/index.js]: Use something like this?
`topLabels = labels.map(x => x.description).slice(0,3);`

File: pubsub/index.js [hunk 3]
```diff
@@ -68,35 +74,46 @@ const extractInfoFromMessage = (message) => {
...
 };
 
 // Write sender, attachment filename, and download link to a Google Sheet.
-const updateReferenceSheet = async (from, filename) => {
-  let link = `https://storage.cloud.google.com/${BUCKET}/${filename}`;
+const updateReferenceSheet = async (from, filename, topLabels) => {
   await googleSheets.spreadsheets.values.append({
     spreadsheetId: SHEET,
-    range: 'Sheet1!A1:D1',
+    range: 'Sheet1!A1:F1',
```
Comment (ace-n) [inline pubsub/index.js]: Are both of these `range` properties necessary?

If so - can we move this to a separate variable?

File: auth/index.js (hunk 1 above)
Comment (michaelawyu) [reply auth/index.js]: Done.

File: pubsub/index.js (hunk 2 above)
Comment (michaelawyu) [reply pubsub/index.js]: Done.

File: pubsub/index.js (hunk 3 above)
Comment (michaelawyu) [reply pubsub/index.js]: Yeah, those ranges are required I think. I moved it to a separate variable.

**Pull request merged**